pages/docs/concepts/asyncapi-document/adding-operations.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Adding Operations
+weight: 60
+---
+
+In a messaging system, the term "operations" refers to the various methods by which messages are exchanged between participants or components. 
+
+## Operations Features
+
+- Operations describe the behaviors and capabilities of the messaging channels described in the document.
```

Comment:
Be specific to avoid confusion. Document can be ambiguous or confusing for a newbie to comprehend or catch on to. 
```suggestion
- Operations describe the behaviors and capabilities of the messaging channels described in the AsyncAPI document.
```